params/precompile_config_test.go

```diff
@@ -59,6 +59,25 @@ func TestVerifyWithChainConfig(t *testing.T) {
 	assert.ErrorContains(t, err, "disable should be [true]")
 }
 
+func TestVerifyWithChainConfigAtNilTimestamp(t *testing.T) {
+	admins := []common.Address{{1}}
+	baseConfig := *SubnetEVMDefaultChainConfig
+	config := &baseConfig
+	config.PrecompileUpgrade = PrecompileUpgrade{
+		// this does NOT enable the precompile, so it should be upgradeable.
+		TxAllowListConfig: precompile.NewTxAllowListConfig(nil, nil, nil),
+	}
```

Comment:
Should we add `require.False(t, config.IsTxAllowList(common.Big0))`after this, just in case?